src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -116,7 +138,20 @@ SystemInventoryCard.propTypes = {
     fetchInventoryTotal: PropTypes.func,
     inventoryTotalSummary: PropTypes.object,
     inventoryTotalFetchStatus: PropTypes.string,
-    intl: PropTypes.any
+    intl: PropTypes.any,
+    selectedTags: PropTypes.arrayOf(PropTypes.string),
+    workloads: PropTypes.oneOf([
```

Comment:
this looks like a real specific shape, should we abstract it to a constants file so all apps can consume it?

Reply:
Good point!

Reply:
I've made it into complex prop type check so we have easier way of identifying what is wrong.